plugins/alias/src/index.js

```diff
@@ -10,8 +10,18 @@ module.exports = function(context, options) {
           alias: {
             CSS: path.resolve(__dirname, '../../../src/css'),
             components: path.resolve(__dirname, '../../../src/components'),
+            react: path.resolve('./node_modules/react'),
+            img: path.resolve(__dirname, '../../../static/img')
           }
-        }
+        },
+        module: {
+          rules: [
+            {
+              test: /\.svg$/,
+              use: ['@svgr/webpack'],
+            },
+          ],
+        },
```

Comment:
This is included in the latest version of docusaurus, so once we update versions I'll just remove it